lego_port.go

```diff
@@ -179,7 +179,9 @@ func (p *LegoPort) Uevent() (map[string]string, error) {
 }
 
 // ConnectedTo returns a description of the device attached to p in the form
-// {inX,outY}:DEVICE-NAME, where X is in {1-4} and Y is in {A-D}.
+// PLATFORM-ports:{inX,outY}:DEVICE, where X is in {1-4} and Y is in {A-D},
+// or if on a BrickPi3 platform, spi0.1:{SX,MY}:DEVICE, where X is in {1-16}
+// and Y is in {A-P}.
```

Comment:
I'm not sure if you care that the description covers all platforms, but BrickPi an PiStorms don't match the patterns listed here. You could generalize the pattern even further at `{device connection}:{port name}` where device connection is the underlying transport (spi0.1, serial0-0, ev3-ports, evb-ports, pistorms) and port name is the name physically printed on the device (with the "in" and "out" prefix for EV3)

Reply:
Done.